src/recommendationservice/requirements.in

```diff
@@ -1,12 +1,13 @@
 google-api-core==1.23.0
 google-python-cloud-debugger==2.18
+google-cloud-profiler==3.0.7
```

Comment:
Question: Why do we manually upgrade `google-cloud-profiler` as well? Is it because the older version of `google-cloud-profiler` doesn't work with `rsa==4.7`?

Reply:
We will eventually have to bump `google-cloud-profiler` so that it's compatible with v2.x of `google-api-core`
While this isn't completely necessary right now, we will have to bump it in the future
(also google-cloud-profiler v1.1.2 is 2 years old 😅 )